src/ops/validate.rs

```diff
@@ -345,7 +263,7 @@ impl ValidateOp for super::Case {
         children: impl DoubleEndedIterator<Item = (NodeIndex, &'a OpType)>,
     ) -> Result<(), ChildrenValidationError> {
         let sig = self.inner_signature();
-        validate_io_nodes(&sig.input, &sig.output, "Conditional", children)
+        validate_io_nodes(&sig.input, &sig.output, "DataflowParent", children)
```

Comment:
I guess there is no handy way to get a name? "DataflowParent" seems fine if nothing springs to mind

Reply:
we can add a dependency on the `OpName` trait and use that but this function
(and everything subsequently including errors) expects static strings so its
just a big annoying change to make.

Reply:
have gone with `std::any::type_name` instead

Reply:
using `T`! Nice :-)